gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -245,10 +245,19 @@ message TripUpdate {
     // Provides the updated values for the stop time.
     // NOTE: This message is still experimental, and subject to change. It may be formally adopted in the future.
     message StopTimeProperties {
-      // Supports real-time platform assignments. Refers to a stop_id defined in the GTFS stops.txt. This stop_id must
-      // belong to the same station as the stop originally defined in GTFS stop_times.txt.
+      // Supports real-time stop assignments. Refers to a stop_id defined in the GTFS stops.txt.
+      // The new assigned_stop_id should not result in a significantly different trip experience for the end user than
+      // the stop_id defined in GTFS stop_times.txt. In other words, the end user should not view this new stop_id as an
+      // "unusual change" if the new stop was presented within an app without any additional context.
+      // For example, this field is intended to be used for platform assignments by using a stop_id that belongs to the
+      // same station as the stop originally defined in GTFS stop_times.txt.
+      // To assign a stop without providing any real-time arrival or departure predictions, populate this field and set
+      // StopTimeUpdate.schedule_relationship = NO_DATA.
+      // If this field is populated, StopTimeUpdate `stop_sequence` must be populated and StopTimeUpdate `stop_id`
+      // should not be populated. Platform assignments should be reflected in other GTFS-realtime fields as well
```

Comment:
Why shouldn't stop_id be populated?

Reply:
This was based on your feedback that we should move towards using `stop_sequence` as the selector for stops - see https://github.com/google/transit/pull/219#issuecomment-717524976.

Reply:
This language seems to conflict with the language below about using both `stop_id` and `assigned_stop_id` below. I think you could drop the part about "and StopTImeUpdate `stop_id`..." and only keep the part about `stop_sequence` being required?

Reply:
@paulswartz Ah, sorry, my mistake - thanks for point it out. I made the change in the spec language but missed this comment in the .proto file - I've updated it in https://github.com/google/transit/pull/219/commits/81e8804fdf88bd3b9512f9b33616ec45fa6f1340 to match the spec language.

@gcamp Just a heads up that this changed since you already voted.